samples/aspnetapp/deploy-container-to-aci.md

```diff
@@ -63,7 +63,7 @@ You can deploy Windows images with the following command, which includes `--os-t
 az container create --name aspnetapp --image richlander.azurecr.io/aspnetapp --resource-group richlander-containers --ip-address public --os-type Windows
 ```
 
-> Note: Azure Container Instances currently supports Windows Server 2016 images based on Long-Term Servicing Channel (LTSC) versions. Windows Semi-Annual Channel (SAC) releases like 1803 and 1903 are unsupported.
+> Note: Azure Container Instances currently supports Windows Server 2016 images based on Long-Term Servicing Channel (LTSC) versions. Windows Semi-Annual Channel (SAC) releases like 1803, 1903, and 1909 are unsupported.
```

Comment:
I have the same comment as I made in .NET Core.  What do you think about refactoring this to reference the ACI documentation on this topic - https://docs.microsoft.com/en-us/azure/container-instances/container-instances-faq#what-windows-base-os-images-are-supported so that we don't have to maintain the list of supported and unsupported versions.

Reply:
Since that's outside the scope of these changes and I don't want to litter this PR with iterations on unrelated doc changes, I'll log this as a separate issue. For now, I'll list 1809 as supported.